src/jupyterlabcontroller/models/v1/lab.py

```diff
@@ -237,6 +234,36 @@ class UserGroup(CamelCaseModel):
     )
 
 
+class NotebookQuota(BaseModel):
+    """Notebook Aspect quota information for a user."""
```

Comment:
Might want to name this, or explain in the comment, to make it clear that this is the quota for the namespace, which is distinct from CPU/memory requests/limits (but the namespace quota must be at least as large as the CPU/memory limit)

Reply:
This model is copied from Gafaelfawr, which thinks of this as the quota for the entire Notebook Aspect. I think this problem will go away once we are importing the Gafaelfawr models from a separate package. Added to my to-do to clean up how the user metadata is shown in the lab, though, since right now this is a bit awkward.